src/components/Common/BadgeMobile/index.tsx

```diff
@@ -0,0 +1,83 @@
+import { forwardRef } from 'react';
+
+interface BadgeMobileProps {
+  badgeType?:
```

Comment:
We should add an is4k prop instead of having a type for each one.

Also, instead of the styling types like primary or warning, let's instead change this to take a media status as a prop.